resources/js/electron-plugin/src/server/php.ts

```diff
@@ -295,8 +295,8 @@ function getDefaultEnvironmentVariables(secret?: string, apiPort?: number): Envi
         NATIVEPHP_VIDEOS_PATH: getPath('videos'),
         NATIVEPHP_RECENT_PATH: getPath('recent'),
         NATIVEPHP_EXTRAS_PATH: app.isPackaged
-            ? join(app.getAppPath(), 'nativephp', 'extras')
-            : join(app.getAppPath(), '..', '..', 'extras'),
+            ? join(process.resourcesPath, '..', 'extras')
```

Comment:
The original `join(app.getAppPath(), 'nativephp', 'extras')` 
pointed to `playground.app/Contents/Resources/app.asar/nativephp/extras`

Electron builder automatically extracts the extra files in `playground.app/Contents/extras`. 
Using `process.resourcesPath` instead points it to the right place